### PR TITLE
Add postgres tutorial page

### PR DESCRIPTION
**Pull request by kimong1**
* Related issue
  * #144 [UI] - Render Postgres Tutorial MarkDown
  * #97 [Content] Postgresql
* What this PR does
  * Postgres page is added to `/postgres` (Check [here](https://kimong.learndatabases.dev/postgres))
     
  * Databases page has buttons that send users to each database page. Path is changed from `/databases/postgres` to `/postgres` 
  * Previously we didn't separate web page rendering router functions since they are so simple. Now they can be complicated and long so I stored them in `/src/routes/renderRoutes.js`

### CODE REVIEW

**Comment by kimong1** [comment]
It doesn't pass 100% coverage test. I'm still figuring it out

File: views/postgres.ejs [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+<pre><code><span class="hljs-keyword">const</span> { Client } = <span class="hljs-built_in">require</span>(<span class="hljs-string">'pg'</span>)
+
+<span class="hljs-keyword">const</span> startApp = <span class="hljs-keyword">async</span> ()=&gt;{
+    client = <span class="hljs-keyword">new</span> Client({
+            <span class="hljs-attr">host</span>: <span class="hljs-string">'&lt;host name&gt;'</span>,
+                <span class="hljs-attr">port</span>: <span class="hljs-number">5432</span>,
+                    <span class="hljs-attr">user</span>: <span class="hljs-string">'&lt;username&gt;'</span>,
+                        <span class="hljs-attr">password</span>: <span class="hljs-string">'&lt;password&gt;'</span>,
+                            <span class="hljs-attr">database</span>: <span class="hljs-string">'&lt;database_name&gt;'</span>
+                              })
```
Comment (greenc123) [inline views/postgres.ejs:24]: Is there any way you can format this more so it's easier to read?

Comment (kimong1) [reply views/postgres.ejs:24]: I'm considering to use markedjs. I'll update soon

File: src/routes/renderRoutes.js [hunk 2]
```diff
@@ -0,0 +1,18 @@
+const db = require("../../sequelize/db");
+const routes = {};
+
+routes.postgres = async (req, res) => {
+  if (!req.session.username) return res.redirect("/");
+  const { Accounts } = db.getModels();
+  const userAccount = await Accounts.findOne({
+    where: {
+      username: req.session.username,
```
Comment (greenc123) [inline src/routes/renderRoutes.js:9]: Can we look up by id here: `req.session.id`

Comment (kimong1) [reply src/routes/renderRoutes.js:9]: Oh yes

**Pull request merged**